extensions/amp-consent/0.1/amp-consent.js

```diff
@@ -399,30 +405,29 @@ export class AmpConsent extends AMP.BaseElement {
         config['promptIfUnknownForGeoGroup'],
     'neither checkConsentHref nor ' +
     'promptIfUnknownForGeoGroup is defined');
+    let geoPromise = Promise.resolve(null);
     let remoteConfigPromise = Promise.resolve(null);
-    if (config['checkConsentHref']) {
-      remoteConfigPromise = this.getConsentRemote_(instanceId);
-      this.passSharedData_(instanceId, remoteConfigPromise);
-    }
-    let geoPromise = Promise.resolve();
     if (config['promptIfUnknownForGeoGroup']) {
       const geoGroup = config['promptIfUnknownForGeoGroup'];
       geoPromise = this.isConsentRequiredGeo_(geoGroup);
+    } else {
+      remoteConfigPromise = this.getConsentRemote_(instanceId);
     }
+
     return geoPromise.then(promptIfUnknown => {
       return remoteConfigPromise.then(response => {
         this.consentRequired_[instanceId] =
-            this.isPromptRequired_(instanceId, response, promptIfUnknown);
+            this.isPromptRequired_(instanceId, promptIfUnknown, response);
```

Comment:
nit: rename response -> remoteConfigResponse for readability

Reply:
renamed.